guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/organizationalunit/OrganizationalUnitSearchService.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.structure.organizationalunit;
+
+import java.util.Collection;
+
+import org.jboss.errai.bus.server.annotations.Remote;
+
+/**
+ * {@link OrganizationalUnit} remote search interface
+ */
+@Remote
+public interface OrganizationalUnitSearchService {
```

Comment:
too much to use a search service... maybe even all github org units could be loaded directly into a data structure and handled by client side :P